.circleci/config-automatic.yml

```diff
@@ -158,11 +158,13 @@ jobs:
             SERVICE_NAME=<< pipeline.parameters.service_name >>
             cd /tmp/project/Terraform
             TERRAFORM_BACKEND_CONFIG_FILE=<< pipeline.parameters.terraform_backend_config_file >>
-            RESOURCE_GROUP_NAME=`cat /tmp/project/Terraform/$TERRAFORM_ENV_CONFIG_FILE | grep resource_group_name | awk '{print $3}' | sed -e 's/^"//' -e 's/"$//'`
+            APP_PLAN_RESOURCE_GROUP_NAME=`cat /tmp/project/Terraform/$TERRAFORM_ENV_CONFIG_FILE | grep resource_group_name | awk '{print $3}' | sed -e 's/^"//' -e 's/"$//'`
             SERVICE_PLAN_NAME=`cat /tmp/project/Terraform/$TERRAFORM_ENV_CONFIG_FILE | grep service_plan_name | awk '{print $3}' | sed -e 's/^"//' -e 's/"$//'`
             ENVIRONMENT_SUFFIX=`cat /tmp/project/Terraform/$TERRAFORM_ENV_CONFIG_FILE | grep environment_suffix | awk '{print $3}' | sed -e 's/^"//' -e 's/"$//'`
```

Comment:
you might need to use `grep "environment_suffix "` instead of `grep environment_suffix`